airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -315,4 +315,8 @@
   dockerRepository: airbyte/source-amplitude
   dockerImageTag: 0.1.0
   documentationUrl: https://hub.docker.com/r/airbyte/source-amplitude
-# SOURCE DEFINITION BY CODE GENERATOR
```

Comment:
why remove the comment?

Reply:
I think I removed it by accident while resolving a merge conflict. 
Thanks for noticing, I'll bring it back